input/images-source/overview-backend-services.plantuml

```diff
@@ -3,16 +3,16 @@ participant App as "Backend Service"
 participant "FHIR authorization server" as EHR
 participant "FHIR resource server" as FHIR
 
-opt  Precondition: Client Registration
+opt  1. Precondition: Client Registration
```

Comment:
These numbers don't line up vertically and don't make the diagram clearer for me. I'm hesitant to incorporate them here, especially if we're not going to incorporate them into the section titles below.

Reply:
The numbered list should relate to the sequence diagram .. alternate is to renumber the list to a sequentially numbered sequence diagram.

Reply:
We can resolve this at https://github.com/HL7/smart-app-launch/pull/381 and apply our here

Reply:
OK agree that the diagram suffers.